sklearn/utils/tests/test_seq_dataset.py

```diff
@@ -6,17 +6,25 @@
 from numpy.testing import assert_array_equal
 import scipy.sparse as sp
 
-from sklearn.utils.seq_dataset import ArrayDataset, CSRDataset
+from sklearn.utils.seq_dataset import ArrayDataset64 as ArrayDataset
+from sklearn.utils.seq_dataset import CSRDataset64 as CSRDataset
+from sklearn.utils.seq_dataset import ArrayDataset32
+
 from sklearn.datasets import load_iris
 
-from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_equal, assert_array_almost_equal
 
 iris = load_iris()
 X = iris.data.astype(np.float64)
 y = iris.target.astype(np.float64)
 X_csr = sp.csr_matrix(X)
 sample_weight = np.arange(y.size, dtype=np.float64)
 
+X32 = iris.data.astype(np.float32)
+y32 = iris.target.astype(np.float32)
+X_csr32 = sp.csr_matrix(X32)
+sample_weight32 = np.arange(y32.size, dtype=np.float32)
+
 
 def assert_csr_equal(X, Y):
```

Comment:
@glemaitre 	said(https://github.com/scikit-learn/scikit-learn/pull/11155/files#r203940311):
> We should probably test CSRDataset32 as well

Reply:
If any this should be tested in `test_seq_dataset():` but I'm not sure